packages/io/stl-deserializer/tests/translate.test.js

```diff
@@ -89,11 +89,11 @@ const main = () => {
 module.exports = {main}
 `
 
-  const observed = deserializer.deserialize(inputFile, undefined, { output: 'jscad', addMetaData: false })
+  const observed = deserializer.deserialize({ filename: 'ascii', output: 'script', addMetaData: false }, inputFile)
```

Comment:
is the 'ascii' intentional for the filename ?

Reply:
In this case, yes. I just wanted to test a few more codebranches.